### PR TITLE
Added 1.20-4 docs

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/contents/index.md [hunk 1]
```diff
@@ -123,6 +123,7 @@ if you are interested in the AL2 container runtime kernel version.
 * [v1-21-eks-1](releases/1-21/1/index.md) (July 19, 2021)
 
 #### EKS-D 1.20 Version Dependencies
+* [v1-20-eks-4](releases/1-20/4/index.md) (July 26, 2021)
```
Comment (kschumy) [inline docs/contents/index.md:126]:
```suggestion
* [v1-20-eks-4](releases/1-20/4/index.md) (July 27, 2021)
```

**Pull request merged**